### ISSUE TITLE
MCFSRs?

### ISSUE THREAD

**Issue by davidfetter** [whatisnuclear/website#8]
Apparently, there are fast spectrum reactors, or at least designs for same, that use molten salt fuel. Is there enough `there` there to write something up one way or the other?

**Comment by partofthething** [comment]
yeah that's a good idea. [They are mentioned briefly on the MSR page](https://whatisnuclear.com/msr.html) but could be expanded upon

**Comment by davidfetter** [comment]
I don't know quite how to put this gently, but as far as I can tell, LFTR is a scam based on at best speculative extensions of the work done at Oak Ridge with multiple-weapon quantities of weapons grade ²³³U and one experiment with a teensy amount of thorium. Intractable problems other than that ²³³U required in order to get to criticality include:

- A source of large amounts of highly enriched ⁷Li for the FLiBe, production of which also gets you large amounts of enriched ⁶Li, which is weapons material.
- Credible ways to prevent diversion of the extremely pure ²³³Pa, which quickly turns into extremely pure ²³³U, that the thing will produce when fed thorium.
- Steel that remains at best speculative, given the constraint that it can't have Co in it because of activation.
- Automated processing on site that requires really nasty chemistry with fluorine and so is itself a huge chemical hazard.

I don't believe that LFTR has earned a page framed other than as "overwhelmingly likely a scam." It'd be good to have one on molten salt fast reactors, which at least are based on things that have a conceivable supply chain, etc., and if they turn out to be just as scammy, to mention that fact prominently.
